{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```
@@ -1,57 +1,82 @@
+---
 exclude: "^docs/source/conf.py$"
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
-    hooks:
-    -   id: trailing-whitespace
-    -   id: end-of-file-fixer
-    -   id: check-added-large-files
-    -   id: check-executables-have-shebangs
-    -   id: check-case-conflict
-    -   id: check-vcs-permalinks
-    -   id: forbid-new-submodules
-    -   id: mixed-line-ending
-    -   id: check-merge-conflict
-    -   id: detect-private-key
-    -   id: detect-aws-credentials
-        args: [ '--allow-missing-credentials']  # Avoid failure on CI
-    -   id: check-toml
-    -   id: check-yaml
-    -   id: check-json
--   repo: https://github.com/shellcheck-py/shellcheck-py
-    rev: v0.8.0.4
-    hooks:
-    -   id: shellcheck
-        name: Shellcheck
-        args: ["-f", "gcc"]  # output filename:linenum:colnum (clickable)
--   repo: https://github.com/AleksaC/hadolint-py
-    rev: v2.10.0
-    hooks:
-      - id: hadolint
-        name: Hadolint (Dockerfile checker)
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.4.0
+  hooks:
+  - id: trailing-whitespace
+  - id: end-of-file-fixer
+  - id: check-added-large-files
+  - id: check-executables-have-shebangs
+  - id: check-case-conflict
+  - id: check-vcs-permalinks
+  - id: forbid-new-submodules
+  - id: mixed-line-ending
+  - id: check-merge-conflict
+  - id: detect-private-key
+  - id: detect-aws-credentials
+    args: ['--allow-missing-credentials']  # Avoid failure on CI
+  - id: check-toml
+  - id: check-yaml
+  - id: check-json
+- repo: https://github.com/shellcheck-py/shellcheck-py
+  rev: v0.8.0.4
+  hooks:
+  - id: shellcheck
+    name: Shellcheck
+    args: ["-f", "gcc"]  # output filename:linenum:colnum (clickable)
+- repo: https://github.com/AleksaC/hadolint-py
+  rev: v2.10.0
+  hooks:
+  - id: hadolint
+    name: Hadolint (Dockerfile checker)
 # Actual Python Linters
--   repo: https://github.com/psf/black
-    rev: 22.10.0
-    hooks:
-      - id: black
-        name: Black (Python formatter)
--   repo: https://github.com/pycqa/isort
-    rev: 5.10.1
-    hooks:
-      - id: isort
-        name: isort (Python import sorter)
--   repo: https://github.com/pycqa/flake8
-    rev: 5.0.4
-    hooks:
-    -   id: flake8
-        name: Flake8 (Python linter)
-        args: ["--output-file", "test_results/flake8.txt", "--tee", "src/", "tests/"]
-        additional_dependencies:
-          - "flake8-bandit==4.1.1"
-          - "flake8-bugbear==22.8.23"
-          - "flake8-docstrings==1.6.0"
--   repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.982
-    hooks:
-    -   id: mypy
-        name: Mypy (Python type-checker)
+- repo: https://github.com/psf/black
+  rev: 22.12.0
+  hooks:
+  - id: black
+    name: Black (Python formatter)
+- repo: https://github.com/pycqa/isort
+  rev: v5.11.3
```

Since we last talked, isort/poetry interacted into breaking the world's CI, which requires bumping our packages everywhere (not linking the actual ticket to avoid xref annoyances, but you can see in isort issue number 2078).

```suggestion
  rev: 5.12.0
```

---

I wasn't able to reproduce the problems mentioned in the ticket, but I'm sure you've had to deal with the issues more than enough so I'll take your word that the update is necessary.